maven-scm-test/pom.xml

```diff
@@ -60,7 +60,6 @@
       <groupId>com.google.inject</groupId>
       <artifactId>guice</artifactId>
       <classifier>no_aop</classifier>
-      <scope>runtime</scope>
```

Comment:
Is this related to dropping JUnit 3?

Reply:
yes, as now Guice classes ARE used in compile scope in maven-scm-test, see Robert's PlexusJunit4TestSupport class, it has import `import com.google.inject.Module;`

Reply:
OTOH, this module is ONLY in test scope of other modules, so guice is compile to this module, while is test + runtime for  others